modules/neovim/nixvim.nix

```diff
@@ -92,8 +22,125 @@ in
       signColumn = lib.mkEnableOption "background transparency for the NeoVim sign column";
       numberLine = lib.mkEnableOption "background transparency for the NeoVim number/relativenumber column";
     };
+
+    exportedModule = lib.mkOption {
+      type = lib.types.attrs;
+      description = ''
+        Theming configuration which can be merged with your own. See
+        [Standalone Mode](#standalone-mode) documentation.
+      '';
+      default = { };
+    };
+    pluginConfigs = lib.mkOption {
+      type = lib.types.attrs;
+      default = {
+        "base16-nvim".colorschemes.enable = true;
+        "mini.base16".plugins.mini.enable = true;
+      };
+      internal = true;
+    };
```

Comment:
> NixVim theming is completely nonfunctional, and I'm not sure why.

Is the `default` attribute set that enables the plugins fully overriden by:

https://github.com/nix-community/stylix/blob/ebf82e070861cf21a65edf2ccf2fc51d58e29612/modules/neovim/nixvim.nix#L48

resulting in the `default` being lost?